bluecellulab/cell/template.py

```diff
@@ -76,14 +76,19 @@ def get_cell(
             cell = getattr(neuron.h, self.template_name)(
                 gid, morph_dir, morph_fname
             )
-        elif template_format == "v6_ais_scaler":
+        elif template_format == "v6_adapted":
             if emodel_properties is None:
                 raise BluecellulabError(
                     "EmodelProperties must be provided for template "
-                    "format v6_ais_scaler"
+                    "format v6_adapted"
                 )
+
             cell = getattr(neuron.h, self.template_name)(
-                gid, morph_dir, morph_fname, emodel_properties.ais_scaler
+                gid,
+                morph_dir,
+                morph_fname,
+                emodel_properties.ais_scaler,
+                emodel_properties.soma_scaler
```

Comment:
@anilbey , it is used here

Reply:
Ah yes. True. Where is the `soma_scaler` set? Is it to be set by the user?

For the ais_scaler there is this code to read it from the circuit. Can the soma_scaler also be read from the circuit in a similar way?

```python
        if "@dynamics:AIS_scaler" in self.available_cell_properties:
            emodel_properties.ais_scaler = float(self.get_cell_properties(
                cell_id, properties=["@dynamics:AIS_scaler"])["@dynamics:AIS_scaler"])
```

https://github.com/BlueBrain/BlueCelluLab/blob/e4c089567b5e309b96450e21398a23c86605cf66/bluecellulab/circuit/circuit_access.py#L216C1-L219C1

Reply:
yes, the same, I have updated this part as well